Задание6.2/Task62.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+
+namespace Задание6._2
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            var x = double.Parse(Console.ReadLine());
+            var y = double.Parse(Console.ReadLine());
+            static bool НазваниеМетода(double x, double y)
+            {
+                return ((x <= -2) && (y >= 1));
+            }
+            Console.WriteLine(НазваниеМетода(x, y));
+            Console.ReadKey();
+        }
+    }
```

Comment:
Старайтесь без особых причин не использовать метод в методе. И название нормальное можно было бы придумать.
```suggestion
    class Program
    {
        static void Main(string[] args)
        {
            var x = double.Parse(Console.ReadLine());
            var y = double.Parse(Console.ReadLine());

            Console.WriteLine(НазваниеМетода(x, y));
            Console.ReadKey();
        }
        
            static bool НазваниеМетода(double x, double y)
            {
                return ((x <= -2) && (y >= 1));
            }
    }
```